### PR TITLE
fix: spacing issue anonymous rank modal

### PR DESCRIPTION
**Pull request by Showrin**
### Describe what this PR does
- There was no gap between the signup button and the RanksTags section. This PR fixes this inconsistency.

image

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [x] Have you done sanity checks in the webapp?
- [x] Have you done sanity checks in the extension?
- [x] Does this not break anything in companion?

### Did you test the modified components media queries?
- [x] MobileL (420px)
- [x] Tablet (656px)
- [x] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

File: packages/shared/src/components/modals/RanksModal/index.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ export default function RanksModal({
           previousRank={previousRank}
         />
         {!user && (
-          <div className="flex flex-col items-center mt-2">
+          <div className="flex flex-col items-center mt-2 mb-3">
```
Comment (rebelchris) [inline packages/shared/src/components/modals/RanksModal/index.tsx:47]:
```suggestion
          <div className="flex flex-col items-center mt-2 mb-7">
```

**Review by rebelchris — CHANGES_REQUESTED** [review]
I've checked with our design team, and they would like to make it 28 pixels between the button and the bottom part.

If you could change it to my suggestion we can approve and continue.
Thanks for the action on this.

Screenshot 2023-04-21 at 11 25 27

File: packages/shared/src/components/modals/RanksModal/index.tsx (hunk 1 above)
Comment (Showrin) [reply packages/shared/src/components/modals/RanksModal/index.tsx]: @rebelchris Thanks for the feedback. I've updated the spacing. 😊

**Review by rebelchris — APPROVED** [review]
Amazing, thanks for this work 🙏

**Pull request merged**